Signal/src/views/OWSSystemMessageCell.m

```diff
@@ -265,9 +293,12 @@ - (CGSize)cellSizeForInteraction:(TSInteraction *)interaction collectionViewWidt
     [self applyTitleForInteraction:interaction label:self.titleLabel];
     CGFloat maxTitleWidth = (collectionViewWidth - ([self hMargin] * 2.f + [self hSpacing] + [self iconSize]));
     CGSize titleSize = [self.titleLabel sizeThatFits:CGSizeMake(maxTitleWidth, CGFLOAT_MAX)];
+
     CGFloat contentHeight = ceil(MAX([self iconSize], titleSize.height));
     result.height += contentHeight;
 
+    result.height += [self topLabelBottomMargin];
```

Comment:
You're right. I think I've pushed up a better approach here:
https://github.com/WhisperSystems/Signal-iOS/pull/2231/commits/5a0d6cba581cbe248540eb0f11535179cacfce5a

PTAL @charlesmchen